lib/tasks/locale.rake

```diff
@@ -141,12 +141,18 @@ namespace :locale do
     system({"RAILS_ENV" => "i18n"}, "bundle exec rake locale:store_model_attributes")
   end
 
+  task "delete_pot_file", :root do |_, args|
+    pot_file = Dir.glob("#{args[:root]}/locale/*.pot").first
```

Comment:
Since the arg is a Pathname, this is probably preferable: `args[:root].join("locale/*.pot")`

Reply:
`args[:root]` is not a `Pathname` object, but we can make it one. Fixed.